{{cookiecutter.project_name}}/.github/workflows/tests.yml

```diff
@@ -81,10 +81,10 @@ jobs:
           nox --force-color --python=${{ "{{" }} matrix.python-version {{ "}}" }}
 
       - name: Upload documentation
-        if: matrix.session == 'docs'
+        if: matrix.session == 'docs-build'
         uses: actions/upload-artifact@v2
         with:
-          name: docs
+          name: docs-build
```

Comment:
Let's not change the artifact name. This can stay plain `docs`.

Reply:
Done.